### DISCUSSION TITLE
SSR "Window is not defined" apexcharts

### DISCUSSION THREAD

**Discussion by leoMirandaa** [quasarframework/quasar#11195] [CLI - SSR mode] [answered]
Hi there!
I'm workin with Quasar-SSR and Vue V2 but i have a problem when i'm trying to use apexchart/vueApexchart on SSR. 
what i did:
- installed apexcharts, vue-apexcharts
- Create a file into boot folder > apexcharts.js > `import Vue from 'vue'
import VueApexCharts from 'vue-apexcharts' 
Vue.component('apexchart', VueApexCharts)`
- on  charts.vue i use tag < apexchart > and their props etc..
- add apexchart on quasar.conf.js > boot: ['apexchart']

when i use 'quasar dev' the app works fine and show the chart correctly but.. when i use 'quasar dev -m ssr' it shows "window is not defined" ....node_modules\apexcharts\dist\apexcharts.min.js

other thing that i was trying is create a plugins folder> apexchart.js > `import Vue from 'vue'
import VueApexCharts from 'vue-apexcharts' 
Vue.component('apexchart', VueApexCharts)`  and add on quasar.conf.js >` boot: { path: '~plugins/apexcharts', server: false }`,but i failed
some one know how to configure and use apexcharts on Quasar SSR?.

**Comment by metalsadman** [comment, marked as answer]
https://v1.quasar.dev/quasar-cli/boot-files#usage-of-boot-files
```js
boot: [
  { path: '/src/plugins/apexcharts.js', server: false },
   ... // other boot
]
```
https://apexcharts.com/docs/vue-charts/
```js
// src/plugins/apexcharts.js
import VueApexCharts from 'vue-apexcharts'
Vue.use(VueApexCharts)
Vue.component('apexchart', VueApexCharts)
```

**Reply by leoMirandaa** [reply]
That works correcly, thank you a lot!
